test/Experimental/basisLieHighestWeight/MBOld.jl

```diff
@@ -0,0 +1,389 @@
+ 
+
+
+module MBOld
+
+export basisLieHighestWeight # use parallel = true for parallel computing
+
+using Oscar
+
+
+G = Oscar.GAP.Globals
+forGap = Oscar.GAP.julia_to_gap
+fromGap = Oscar.GAP.gap_to_julia
+
+#### vector space bases
+
+TVec = SRow{ZZRingElem} 
+Short = UInt8 
+
+struct VSBasis
+    A::Vector{TVec} 
+    pivot::Vector{Int} 
+end
+
+
+nullSpace() = VSBasis([], [])
+
+
+function normalize(v::TVec)
+    """
+    divides vector by gcd of nonzero entries, returns vector and first nonzero index
+    used: addAndReduce!
+    """
+    if isempty(v)
+        return (0, 0)
+    end
+
+    pivot = first(v)[1]
+
+    return divexact(v, gcd(map(y->y[2], union(v)))), pivot
+end
+
+
+reduceCol(a, b, i::Int) = b[i]*a - a[i]*b
+
+
+function addAndReduce!(sp::VSBasis, v::TVec)
+    """
+    for each pivot of sp.A we make entry of v zero and return the result
+    0 => linear dependent
+    * => linear independent, new column element of sp.A since it increases basis
+    invariants: the row of a pivotelement in any column in A is 0 (except the pivotelement)
+               elements of A are integers, gcd of each column is 1
+    """
+    A = sp.A
+    pivot = sp.pivot
+    v, newPivot = normalize(v) 
+    if newPivot == 0
+        return 0
+    end
+ 
+    for j = 1:length(A)
+        i = pivot[j]
+        if i != newPivot
+            continue
+        end
+        v = reduceCol(v, A[j], i)
+        v, newPivot = normalize(v)
+        if newPivot == 0
+            return 0
+        end
+    end
+
+    pos = findfirst(pivot .> newPivot)
+    if (pos === nothing)
+        pos = length(pivot) + 1
+    end
+
+    insert!(A, pos, v)
+    insert!(pivot, pos, newPivot)
+
+    return v
+end
+
+
+#### Lie algebras
+
+G = Oscar.GAP.Globals
+forGap = Oscar.GAP.julia_to_gap
+fromGap = Oscar.GAP.gap_to_julia
```

Comment:
Just to say, there are still multiple copies of the above three "aliases", and as I pointed out before, they should all go, as should all uses of `Oscar.GAP.julia_to_gap
` and `Oscar.GAP.gap_to_julia`. But I am guessing you have this on your TODO and just did not yet get to it.